modules/admin-portal/pages/monitoring/log-viewer.adoc

```diff
@@ -57,4 +57,33 @@ image:single-search-result.png[image]
 
 Clicking on a single search result will show the log content in the display panel and highlight the matching log line.
 
-image:line-highlight-log (1).png[image]
\ No newline at end of file
+image:line-highlight-log (1).png[image]
+
+== Read-Only Access to Logs
+
+To streamline the debugging and optimization process, *read-only access to logs* has been introduced for application developers. This feature allows developers to independently view and monitor logs through *GraphStudio* or the *TigerGraph Cloud Portal* without requiring sysadmin intervention.
+
+=== Privileges
+
+==== APP_ACCESS_LOG Privilege
+The `APP_ACCESS_LOG` privilege provides *global access* to logs for authorized users.
+* *Scope:* Global
```

Comment:
How come this privilege has these 3 bullet items (scope, default roles and purpose), but the next one doesn't?

Let's think about describing *privileges* the same way throughout the entire spec.  Other privileges are summarized in this table: https://docs.tigergraph.com/tigergraph-server/4.1/user-access/rbac-row-policy/row-policy-privileges-table (I think the table isn't yet perfect, but at least it's consistent).